cmd/shannon.go

```diff
@@ -23,10 +23,7 @@ func getShannonFullNode(logger polylog.Logger, config shannon.FullNodeConfig) (s
 		return lazyFullNode, nil
 	}
 
-	logger.Warn().Msgf("Invalid Shannon full node config: %v", config)
-
-	// TODO_MVP(@adshmh): add the required logic to initialize a caching full node once it is implemented.
-	return nil, fmt.Errorf("LazyMode is the only supported mode.")
+	return shannon.NewCachingFullNode(logger, lazyFullNode), nil
```

Comment:
@commoddity I'm going to pre-ematively review this, but can you please so a grep for all `lazyFullNode`, `lazy_mode`, and other permutations to update the docs/configs/tooling?

E.g. https://github.com/search?q=repo%3Abuildwithgrove%2Fpath%20lazy_mode&type=code

![Screenshot 2025-04-29 at 2 08 50 PM](https://github.com/user-attachments/assets/8b0e20df-ed00-4d64-818c-c87de5553302)

Reply:
NVM. Did it myself.